inference-engine/src/hetero_plugin/hetero_executable_network.cpp

```diff
@@ -908,6 +907,11 @@ void HeteroExecutableNetwork::ExportImpl(std::ostream& heteroModel) {
             subnetwork._network.Export(heteroModel);
         } catch (InferenceEngine::details::InferenceEngineException& ie_ex) {
             if (std::string::npos != std::string{ie_ex.what()}.find(NOT_IMPLEMENTED_str)) {
+                // TODO: enable once serialization to IR v10 is implemented
+#if 1
```

Comment:
Do we really need this define?

Reply:
why not? Just want to denote that this code should be removed (under 1) and `else` branch should be used once true serialization is implemented.

Reply:
Ok, looks good.